xml/System.Net/HttpListenerResponse.xml

```diff
@@ -159,7 +159,7 @@
  -or-  
   
  <paramref name="name" /> or <paramref name="value" /> contains invalid characters.</exception>
-        <exception cref="T:System.ArgumentOutOfRangeException">The length of <paramref name="value" /> is greater than 65,535 characters.</exception>
+        <exception cref="T:System.ArgumentOutOfRangeException">.NET Framework and .NET Core only: The length of <paramref name="value" /> is greater than 65,535 characters.</exception>
```

Comment:
```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException">.NET Framework and .NET Core versions 1.0 - 3.1 only: The length of <paramref name="value" /> is greater than 65,535 characters.</exception>
```

Since .NET 5 is technically .NET Core as well, I think we need to put an upper version limit. Could you make this change throughout the file?

Reply:
Fixed.